test/go.test.ts

```diff
@@ -19,19 +19,25 @@ import { testCurrentFile } from '../src/goTest';
 import { getGoVersion, isVendorSupported } from '../src/util';
 import { documentSymbols } from '../src/goOutline';
 import { listPackages } from '../src/goImport';
-import { generateTestCurrentFile, generateTestCurrentPackage } from '../src/goGenerateTests';
+import { generateTestCurrentFile, generateTestCurrentPackage, generateTestCurrentFunction } from '../src/goGenerateTests';
 import { getBinPath } from '../src/goPath';
 
 suite('Go Extension Tests', () => {
 	let gopath = process.env['GOPATH'];
 	let repoPath = path.join(gopath, 'src', 'test');
 	let fixturePath = path.join(repoPath, 'testfixture');
 	let fixtureSourcePath = path.join(__dirname, '..', '..', 'test', 'fixtures');
+	let generateTestsSourcePath = path.join(repoPath, 'generatetests');
```

Comment:
if the tests use the same file, 2 tests will failed, since the all tests will be already generated by the "generate test in current package"